src/emqx_plugins.erl

```diff
@@ -228,10 +228,26 @@ with_loaded_file(File, SuccFun) ->
     end.
 
 filter_plugins(Names) ->
-    lists:filtermap(fun(Name1) when is_atom(Name1) -> {true, Name1};
-                       ({Name1, true}) -> {true, Name1};
-                       ({_Name1, false}) -> false
-                    end, Names).
+    Filter =
```

Comment:
Collect a name => boolean map, and return all keys.

please check where the enable/disable logic add new items to the list.
If it’s append, collect with foldl ,
Otherwise foldr 

Reason: a `false` may override a `true` tuple

Reply:
ok, fixed